src/magento/change-log.md

```diff
@@ -4,6 +4,16 @@ title: Change Log
 
 Our documentation is continually updated with new topics, clarifications, and corrections to existing content. Learn more about new features, major updates, and releases, organized by month and year. Check back every now and then to see what’s new.
 
+## March 2020
+
+### Updated topics
+
+|Topic |Change|
+|--- |--- |
+|[Layout Updates]({% link design/layout-updates.md %})|Updated all topics in this section to reflect the change in layout update implementation. Layout updates are now saved on the server and selected from a list. For help composing layout update code, see the [Layout](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/layout-overview.html) section in the Frontend Developer Guide.|
```

Comment:
Let's leave out the sentence about DevDocs. Just a short description of what is updated here.

Reply:
Fixed.